src/script/interpreter.cpp

```diff
@@ -342,6 +342,35 @@ class ConditionStack {
 };
 }
 
+/** Helper for OP_CHECKSIG and OP_CHECKSIGVERIFY
+ *
+ * A return value of false means the script fails entirely. When true is returned, the
+ * fSuccess variable indicates whether the signature check itself succeeded.
+ */
+static bool EvalChecksig(const valtype& vchSig, const valtype& vchPubKey, CScript::const_iterator pend, CScript::const_iterator pbegincodehash, unsigned int flags, const BaseSignatureChecker& checker, SigVersion sigversion, ScriptError* serror, bool& fSuccess)
```

Comment:
nit: Could swap the parameters `pend` and `pbegincodehash`? It's IMHO quite unusual to pass iterator pairs in the order of (end, start) instead of (start, end).

Reply:
This seems reasonable. @sipa - any reason these are reversed?

Reply:
No reason at all.